android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/SubmitPinUseCase.kt

```diff
@@ -0,0 +1,52 @@
+package com.onegini.mobile.sdk.reactnative.clean.use_cases
+
+import com.facebook.react.bridge.Promise
+import com.onegini.mobile.sdk.reactnative.Constants
+import com.onegini.mobile.sdk.reactnative.exception.OneginiReactNativeException
+import com.onegini.mobile.sdk.reactnative.exception.OneginiWrapperErrors.AUTHENTICATION_NOT_IN_PROGRESS
+import com.onegini.mobile.sdk.reactnative.exception.OneginiWrapperErrors.REGISTRATION_NOT_IN_PROGRESS
+import com.onegini.mobile.sdk.reactnative.handlers.pins.CreatePinRequestHandler
+import com.onegini.mobile.sdk.reactnative.handlers.pins.PinAuthenticationRequestHandler
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class SubmitPinUseCase @Inject constructor(
+    private val createPinRequestHandler: CreatePinRequestHandler,
+    private val pinAuthenticationRequestHandler: PinAuthenticationRequestHandler
+) {
+    operator fun invoke(pinFlow: String?, pin: String, promise: Promise) {
```

Comment:
pinFlow: String? -> does it has to be nullable? If yes, you should cover null scenario in `when` statement

Reply:
You are right, I have added null check + check for the actual string and covered that in testCase as well.